level_zero/core/test/unit_tests/gen_kernel.cmake

```diff
@@ -6,18 +6,6 @@
 
 function(level_zero_gen_kernels target platform_name suffix options)
 
-  if(NOT DEFINED cloc_cmd_prefix)
```

Comment:
could you pass `-Dcloc_cmd_prefix=ocloc` to cmake command in your environment instead of removing this code?

Reply:
The problem is that with two or more levels of variable indirection, `cmake` doesn't apply the `CMAKE_CROSSCOMPILING_EMULATOR` wrapper command and this is the case with `cloc_cmd_prefix`.

Reply:
@zboszor  could you verify if it is still needed? cross-build with qemu works fine for us